src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (robot[key] in invertedRobot) {
```

Comment:
The check for property presence is correct, but it's worth noting that using 'in' will check the entire prototype chain. If you want to check only the object's own properties, use 'robot.hasOwnProperty(key)'.